images/builder/Dockerfile

```diff
@@ -0,0 +1,128 @@
+# Copyright 2019 Pusher Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#######################################################################
+###
+### GOLANG BUILDER - For building any go binaries needed in later steps
+###
+#######################################################################
+
+FROM golang:1.12 AS builder
+COPY barnacle/* $GOPATH/src/barnacle/
+
+# RUN apt-get install -y --no-install-recommends ca-certificates git
```

Comment:
I'm assuming this comment can be removed?

Reply:
Erm, yes!